v2/js/screens/settings-aboutcomputer.js

```diff
@@ -32,6 +54,13 @@ const AboutComputer = {
 						<div class="computer-value" >{{userAgent}}</div>
 						<div class="computer-storage" >{{$t('Storage')}}</div>
 						<div class="computer-value" >{{Storage}}</div>
+						<div v-if="clickCount <= 0">
+							<input class="computer-reinit" type="checkbox" v-model="isChecked">
+							<span class="computer-value">{{$t('ReinitJournalAndSettings')}}</span><br/>
+							<div class="computer-value" style="opacity: .7" v-show="isChecked">
+								<span>⚠️</span><span>{{ $t('AllDataWillBeLost') }}</span>
```

Comment:
I'm not a big fan of using an emoji here:

![image](https://github.com/user-attachments/assets/26ced73d-d263-4c3c-aa13-6f69b123d85f)

I suggest to use the warning icon like in v1:

![image](https://github.com/user-attachments/assets/7913255e-dce7-476e-9f1e-f11ab74e468c)

Reply:
✅ Done